packages/bedrock-components/src/index.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
     <title>Bedrock Components</title>
     <script src="https://fast.ssqt.io/npm/@samsquatch/vanilla-components@latest/dist/widget-components.js"></script>
-    <script src="https://fast.ssqt.io/npm/@saasquatch/mint-components/dist/mint-components/mint-components.js"></script>
+    <script src="https://fast.ssqt.io/npm/@saasquatch/mint-components@next/dist/mint-components/mint-components.js"></script>
```

Comment:
Making a note that we will want to swap this back before launch

Reply:
ah meant to correct that, good catch